src/Store/Features.re

```diff
@@ -101,6 +105,28 @@ let update = (~extHostClient, ~configFile, state: State.t, action: Actions.t) =>
     | _ => (state, Effect.none)
     }
 
+  // TODO: This should live in the editor feature project
+  | EditorFont(Service_Font.FontLoaded(font)) => (
+      {...state, editorFont: font},
+      Isolinear.Effect.none,
+    )
+  | EditorFont(Service_Font.FontLoadError(message)) => (
+      state,
+      Feature_Notification.Effects.create(~kind=Error, message)
+      |> Isolinear.Effect.map(msg => Actions.Notification(msg)),
+    )
+
+  // TODO: This should live in the terminal feature project
+  | TerminalFont(Service_Font.FontLoaded(font)) => (
+      {...state, terminalFont: font},
+      Isolinear.Effect.none,
+    )
+  | TerminalFont(Service_Font.FontLoadError(message)) => (
+      state,
+      Feature_Notification.Effects.create(~kind=Error, message)
+      |> Isolinear.Effect.map(msg => Actions.Notification(msg)),
+    )
+
```

Comment:
The terminal font should probably be part of the `Feature_Terminal` module, but that requires first moving the terminal view to the feature project. When that's done, however, this should be moved there.

Same with `EditorFont` above, but that's a longer road.